app/components-react/shared/inputs/SliderInput.tsx

```diff
@@ -36,7 +36,7 @@ export const SliderInput = InputComponent((partialProps: TSliderInputProps) => {
     <InputWrapper {...wrapperAttrs}>
       <Row>
         <Col flex="auto" {...dataAttrs} data-role="input" data-value={inputAttrs.value}>
-          <Slider {...inputAttrs} />
+          <Slider {...inputAttrs} onChange={onChangeHandler} />
```

Comment:
what is this change for?

Reply:
leftover from my divisor prop, will remove